#Parameters for Admittance Controller in user mode
final_vel_topic: /cmd_vel
human_wrench_topic: /human_wrench
update_params_service: /update_parameters
control_parameters:
  rate: 100
  mode: user
  mass: 8
  ldaming: 10
  inertia: 5
  adamping: 20
angular_vel_limit: 0.5 
linear_vel_limit: 0.3
